### PR TITLE
Rename proof of passport to openpassport

### PR DESCRIPTION
**Pull request by 0xturboblitz**

### CODE REVIEW

File: app/i18n/locales/de/projects/proof-of-passport.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "description": "\nProof of passport is developing tools to let users generate proofs of humanity, nationality and age using their government-issued passport. It unlocks uses cases in proof of unique identity, sybil resistance and selective disclosure of private data.\n",
-  "tldr": "Proof of passport lets you check a passport is valid in zero-knowledge"
+  "description": "\nOpenPassport is developing tools to let users generate proofs of humanity, nationality and age using their government-issued passport. It unlocks uses cases in proof of unique identity, sybil resistance and selective disclosure of private data.\n",
```
Comment (ChialiT) [inline app/i18n/locales/de/projects/proof-of-passport.json:2]: "n"OpenPassport 
is it on purpose or? the n

**Review by ChialiT — COMMENTED** [review]
@0xturboblitz the "n"OpenPassport, is it on purpose or?

**Comment by 0xturboblitz** [comment]
> @0xturboblitz the "n"OpenPassport, is it on purpose or?

Looks like there are line jumps at the beginning and end of OpenPassport's and NFCTap's `description`, not sure if this is on purpose or we should remove them @kalidiagne

File: app/i18n/locales/de/projects/proof-of-passport.json (hunk 1 above)
Comment (thebeyondr) [reply app/i18n/locales/de/projects/proof-of-passport.json:2]: The "\n" is a common way to tell parsers to insert a new line. I'm not sure why this needed here in that case. Also, I can't find any other projects in these folders that use this technique.

**Review by thebeyondr — CHANGES_REQUESTED** [review]
Thanks for the updates to the repo! It looks good so far.. If you can remove the "\n" from the copy in locales I'd be happy to approve this 🫡

**Comment by 0xturboblitz** [comment]
Thx @thebeyondr
Just removed the `/n` in descriptions, not sure what they were here for. Seems to work well without it.

**Review by ChialiT — APPROVED** [review]

**Review by thebeyondr — APPROVED** [review]

**Pull request merged**